src/switch_core_media.c

```diff
@@ -5523,6 +5525,11 @@ SWITCH_DECLARE(uint8_t) switch_core_media_validate_common_audio_sdp(switch_core_
 		}
 	}
 
+	if (no_audio) {
+		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "No Audio found! Skip audio validation.\n");
+		return -1;
```

Comment:
Return type is unsigned, this will overflow to 255.

Reply:
Good catch, fix it